dateutil/test/test_parser.py

```diff
@@ -27,6 +27,132 @@
 except ValueError:
     PLATFORM_HAS_DASH_D = False
 
+parser_test_cases = [
```

Comment:
For these sorts of globals I have tended to use ALL_CAPS

Reply:
ok, i'm not sure how i fee about the variable name anyway. All of these tests are "parser test cases"

Reply:
They don't have to go into a variable at all if you don't want.

Most of the time I just pass them directly into the `parametrize` decorator, like so:

```python
@pytest.mark.parametrize('arg1, arg2', [
    (1, 2),
    (3, 4),
    (5, 6),
])
def test_func(arg1, arg2):
    assert func(arg1, arg2)

Reply:
I saw this style of parameterization in some of the other test files. I chose against it because a lot of the important code, like: `@pytest.mark.parametrize('arg1, arg2', [`, is not close to the function being decorated. 

I didn't like that it did not fit on one screen for both how the args are being used and what the args are. With them separated, you have to the data to be injected separate form how it is injected. I liked this because I felt the "how" should live near the function being decorated.

Reply:
Yeah, it's a reasonable choice. I would probably go with ALL_CAPS if you are going with global scope, though. See `dateutil/tests/test_isoparser.py` for how I did it.